README.md

```diff
@@ -149,6 +149,31 @@ These tenets help guide the development of the Guard DSL:
 
 ### Installation
 
+#### Installation from Pre-Built Release Binaries
+
+##### MacOS
+
+By default this is builds for macOS-10 (Catalina). It has been tested to work on macOS-11 (BigSpur). See [OS Matix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners)
+
+1. Open terminal of your choice. Default `Cmd+Space`, type `terminal`
+2. Cut-n-paste the commands below (change version=X for other versions)
+```bash
+     curl -fsSLI -o /dev/null \
```

Comment:
Could just not specify an output file for these instead, right?
```suggestion
     curl -fsSLI \
```

Reply:
@PatMyron 
-I == --head (this is HEAD request)
-L == Location header (which is what the redirect 302 response provides)
These commands worked on both Ubuntu and MacOS that others tested with. Can you please update with what errors you ran into? 

man curl

Reply:
doesn't make sense to pipe head request into being executed, but looks like both `-I` and `-o` got dropped from the README in the last force push 👍